assets/src/pages/room/hooks/useMembraneClient.tsx

```diff
@@ -66,6 +66,11 @@ export const useMembraneClient = (
             }))
           );
         },
+        onTrackAdded: (ctx) => {
+          if (!ctx?.peer) return;
+          const metadata: TrackMetadata = parseMetadata(ctx);
+          api.addTrack(ctx.peer.id, ctx.trackId, undefined, undefined, metadata);
```

Comment:
Could you add a comment here why you're calling `addTrack` with MediaStream and MediaStreamTrack as undefined?

Reply:
Fixed!

Reply:
TBH this seems hacky and I'd rather have some other method in the API to handle this case. I'll merge it for now, but please create an issue to get back to this and solve it without using `undefined`